src/style.css

```
@@ -1,3 +1,109 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap';
+
+:root {
+  --blue-color: #00acdc;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
 body {
+  margin: 0 auto;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  background-color: #fff;
+}
+
+header {
```

Use only classes selector, fix it everywhere
```suggestion
.header {
```